application/frontend/config/main.php

```diff
@@ -1,32 +1,43 @@
 <?php
 
+use Sil\Idp\IdSync\common\components\ApiConsumer;
```

Comment:
no biggie, I'll get to it later: https://github.com/silinternational/idp-id-broker/issues/40